eodag/plugins/download/aws.py

```diff
@@ -705,191 +710,152 @@ def _stream_download_dict(
         self,
         product: EOProduct,
         auth: Optional[Union[AuthBase, S3SessionKwargs]] = None,
-        progress_callback: Optional[ProgressCallback] = None,
+        byte_range: tuple[Optional[int], Optional[int]] = (None, None),
+        compress: Literal["zip", "raw", "auto"] = "auto",
         wait: float = DEFAULT_DOWNLOAD_WAIT,
         timeout: float = DEFAULT_DOWNLOAD_TIMEOUT,
-        **kwargs: Unpack[DownloadConf],
+        **kwargs: Any,
```

Comment:
why changing this ?

Reply:
I just forgot to put it back to it's original value. I'll fix it